tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -759,7 +759,6 @@ node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighth
   Type 'number' is not assignable to type 'void'.
 node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(19744,7): error TS2339: Property 'expected' does not exist on type 'Error'.
 node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(20005,8): error TS2339: Property 'runLighthouseForConnection' does not exist on type 'Window'.
-node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(20015,1): error TS2554: Expected 0 arguments, but got 1.
```

Comment:
Seems like a mistake: updateBadgeFn is initialised with `function() { }`, then called twice as `updateBadgeFn()` and once as `updateBadgeFn(url)`. The 1-parameter call used to be an error but no longer is.

```js
    window.runLighthouseForConnection=function(
        connection,url,options,categoryIDs,
        updateBadgeFn=function(){}){
        const config=options&&options.fastMode?new Config(fastConfig):new Config({
            extends:'lighthouse:default',
            settings:{onlyCategories:categoryIDs}});



        const runOptions=Object.assign({},options,{url,config});
        updateBadgeFn(url); // Here!

        return Runner.run(connection,runOptions).
            then(result=>{
                updateBadgeFn();
                return result;
            }).
            catch(err=>{
                updateBadgeFn();
                throw err;
            });
    };
```